src/lib/index.js

```diff
@@ -190,4 +190,10 @@ if (typeof VERSION !== 'undefined') {
   jsf.VERSION = VERSION;
 }
 
-export default jsf;
+
+// Export an object that has all of jsf()'s properties and is not a function
+// Calling jsf() is deprecated in favor of JSONSchemaFaker.generate() / JSONSchemaFaker.resolve()
+export const JSONSchemaFaker = {... jsf};
```

Comment:
Out of curiosity the object copy is needed? 🤔

Reply:
So that it’s not a function. If you use the named export, calling `JSONSchemaFaker()` is not merely deprecated, it’s not possible.

Reply:
Awesome, TIL about it, thank you!